src/types/reviewRequest.ts

```diff
@@ -48,3 +48,8 @@ export interface ReviewRequest {
   reviewRequestedTime: number
   changedItems: EditedItemProps[]
 }
+
+export interface BlobDiff {
+  old: string
+  new: string
```

Comment:
Same issue with backend, I think we should avoid using the `new` reserved keyword.

Reply:
thanks for spotting, changed!